test/check-session

```diff
@@ -28,25 +28,27 @@ class TestSession(MachineCase):
         # TODO: Make this work with non-root
         b.default_user = "root"
 
-        def check_session(should_exist):
-            check = check_in if should_exist else check_not_in
-            check(m.execute("loginctl list-sessions"), "admin")
+        def wait_session(should_exist):
+            def cond():
+                check = check_in if should_exist else check_not_in
+                return check(m.execute("loginctl list-sessions"), "admin")
+            wait (cond)
 
         def kill_user_admin():
             # logind from systemd 208 is buggy, so we use systemd directly if it fails
             # https://bugs.freedesktop.org/show_bug.cgi?id=71092
             m.execute("loginctl kill-user admin || systemctl kill user-1000.slice")
 
-        check_session(False)
+        wait_session(False)
```

Comment:
Just waiting is a common in all the tests, and we don't try in general to avoid it, so I'd say it is fine here, too.

It only really matters when the test is broken: Without the wait, it will fail 60 seconds earlier.  This really only matters during debugging, and then I usually just reduce the timeout to 10 seconds.

So I'd say we stick with the theme and just wait here, too.